libs/checkpoint-postgres/langgraph/checkpoint/postgres/base.py

```diff
@@ -245,7 +245,7 @@ def get_next_version(self, current: Optional[str], channel: ChannelProtocol) ->
             current_v = int(current.split(".")[0])
         next_v = current_v + 1
         try:
-            next_h = md5(self.serde.dumps_typed(channel.checkpoint())[1]).hexdigest()
+            next_h = sha1(self.serde.dumps_typed(channel.checkpoint())[1]).hexdigest()
```

Comment:
I'm naively assuming since there's more entropy in sha1 it's at least as unlikely to have a collision as md5 ?

Reply:
yea less likely, but this isn't that important here tbh, even sha1 is way more entropy than we'd ever need here